### PR TITLE
Fix container register for gh workflow

### PR DESCRIPTION
**Pull request by mvdbeek**
Continues https://github.com/galaxyproject/planemo/pull/1133

### CODE REVIEW

**Comment by nsoranzo** [comment]
mypy CI build is fixed in https://github.com/galaxyproject/planemo/pull/1134 if you have time to review :wink:

File: planemo/github_util.py [hunk 1]
```diff
@@ -55,12 +56,13 @@ def clone_fork_branch(ctx, target, path, **kwds):
             pass
```
Comment (nsoranzo) [inline planemo/github_util.py:56]: I think you should a `return` to `fork(ctx, path, **kwds)` 2 lines above.

File: planemo/github_util.py [hunk 2]
```diff
@@ -55,12 +56,13 @@ def clone_fork_branch(ctx, target, path, **kwds):
...
 
 
-def fork(ctx, path, **kwds):
+def fork(ctx, path, remote_name=DEFAULT_REMOTE_NAME, **kwds):
     """Fork the target repository using ``hub``."""
     gh_path = ensure_gh(ctx, **kwds)
     gh_env = get_gh_env(ctx, path, **kwds)
-    cmd = [gh_path, "repo", "fork"]
+    cmd = [gh_path, "repo", "fork", '--remote', '--remote-name', remote_name]
```
Comment (nsoranzo) [inline planemo/github_util.py:63]:
```suggestion
    cmd = [gh_path, "repo", "fork", '--remote', 'true', '--remote-name', remote_name]
```

Comment (mvdbeek) [reply planemo/github_util.py:63]: The `remote` flag doesn't take a value in 1.5.

File: planemo/github_util.py (hunk 1 above)
Comment (mvdbeek) [reply planemo/github_util.py:56]: Yep, thanks!

**Comment by mvdbeek** [comment]
Alright, this is working now

**Comment by mvdbeek** [comment]
Thanks @nsoranzo!

**Pull request merged**